staging/src/k8s.io/apiextensions-apiserver/pkg/apiserver/customresource_handler.go

```diff
@@ -172,7 +182,8 @@ func (r *crdHandler) ServeHTTP(w http.ResponseWriter, req *http.Request) {
 		r.delegate.ServeHTTP(w, req)
 		return
 	}
-	if !apiextensions.IsCRDConditionTrue(crd, apiextensions.Established) {
+	if !apiextensions.IsCRDConditionTrue(crd, apiextensions.Installing) ||
```

Comment:
is `||` correct?

Reply:
I think `&&` is the correct one actually. If CRD isn't Established **and** CRD isn't Installing, we're delegating the request, right?

Reply:
agree